guidelines/github-guidelines.md

```diff
@@ -0,0 +1,857 @@
+# GitHub Guidelines
+
+Guidelines, recommendations, and instructions for how to work with FOCUS on GitHub, using tools like a web browser, GitHub Desktop, and VSCode.
+
+## Table of Contents
+
+1. [FOCUS Repository Structure Overview](#focus-repository-structure-overview)
+2. [Logging into GitHub](#logging-into-github)
+3. [Configuring Notifications](#configuring-notifications)
+4. [Installing GitHub Desktop](#installing-github-desktop)
+5. [Installing Visual Studio Code](#installing-visual-studio-code)
+6. [How to Interact (review, comment, participate in, etc.) with an Issue via Web Browser](#how-to-interact-review-comment-participate-in-etc-with-an-issue-via-web-browser)
+7. [Cloning a Repository with GitHub Desktop](#cloning-a-repository-with-github-desktop)
+8. [How to submit a Pull Request (PR) using GitHub Desktop and VSCode](#how-to-submit-a-pull-request-pr-using-github-desktop-and-vscode)
+9. [How to submit changes to an an existing PR using GitHub desktop and vscode](#how-to-submit-changes-to-an-an-existing-pr-using-github-desktop-and-vscode)
+10. [Tips for Success](#tips-for-success)
+
+---
+
+## FOCUS Repository Structure Overview
```

Comment:
@ljadvey. For completeness, this section should also mention two important documents: RELEASE_PLANNING.md and CONTRIBUTING.md. What do you think?